repo/packages/D/dcos-ui/2/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "4.0",
+  "name": "dcos-ui",
+  "version": "2.82.6+0e5782c0",
```

Comment:
we don't want a sha in this version.

Reply:
at least we should test this.

Reply:
CI also complains, yeah… not sure why tho, its valid semver 🤔